### ISSUE TITLE
Training instructions

### ISSUE THREAD

**Issue by willyd** [sabarim/itis#4]
Could you please add the instructions to train a model from the DeepLab weights?

**Comment by Damanpreet** [comment]
Hi @willyd Were you able to train the network? 
I am stuck at trying to train the batch normalization layers? Looks like there is some issue with TensorFlow slim batch norm layers.
